### ISSUE TITLE
Bump

### ISSUE THREAD

**Issue by unfortunatelyalex** [caretbay47/Brawlhalla-AI#2]
Hey, I just wanted to ask if this still is an active project that's been worked on?

**Comment by caretbay47** [comment]
Not really... i can just release what i made for it... but its impossible
to train an ai to play brawlhalla without the source code for brawlhalla

**Comment by unfortunatelyalex** [comment]
Could you please release your progress? I want to see how far you've come and maybe continue it a bit.
thank you :)

**Comment by caretbay47** [comment]
Alright will do

**Comment by unfortunatelyalex** [comment]
bump v2

**Comment by unfortunatelyalex** [comment]
sad bump
